java/bundles/org.eclipse.set.feature.plazmodel/src/org/eclipse/set/feature/plazmodel/check/GeoCoordinateValid.java

```diff
@@ -231,6 +247,40 @@ private static List<Punkt_Objekt> getRelevantPOs(
 				.toList();
 	}
 
+	/**
+	 * @param state
+	 *            the container type
+	 * @param po
+	 *            the {@link Punkt_Objekt}
+	 * @param potk
+	 *            the {@link Punkt_Objekt_TOP_Kante_AttributeGroup} of this
+	 *            {@link Punkt_Objekt}
+	 * @return the {@link GEOKanteCoordinate}
+	 */
+	public GEOKanteCoordinate calculateCoordinate(final ContainerType state,
+			final Punkt_Objekt po,
+			final Punkt_Objekt_TOP_Kante_AttributeGroup potk) {
+		List<TopologicalCoordinate> alreadyDetermine = topologischeCoordinaten
```

Comment:
@TruongQuangSB I would prefer to rename it for better readability

```suggestion
		List<TopologicalCoordinate> alreadyCalulatedCoordinates = topologischeCoordinaten
```